<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-PSP68KB" name="com.Tour.qa.testdata.RegistrationTest" tests="34" failures="3" timestamp="24 Apr 2022 01:51:58 GMT" time="1377.832" errors="0">
  <testcase name="CheckFillDataInRegistrationForm" time="45.765" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <testcase name="CheckInvalidEmail" time="40.535" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <testcase name="CheckduplicateEmail" time="38.685" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <testcase name="CheckinvalidData" time="45.508" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <testcase name="emptyFormtTest" time="40.090" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <testcase name="invalidPasswordTest" time="49.225" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <testcase name="invalidZipCodeTest" time="39.849" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <testcase name="invalidMobNumber" time="42.457" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <testcase name="ErrorShouldDisplyOnPrpviding10CharInMobileFildOdRegidtrationPage" time="43.568" classname="com.Tour.qa.testdata.RegistrationTest">
    <failure type="java.lang.AssertionError" message="expected [The value &#039;qwertyuiop&#039; is invalid.] but found []">
      <![CDATA[java.lang.AssertionError: expected [The value 'qwertyuiop' is invalid.] but found []
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.Tour.qa.testdata.RegistrationTest.ErrorShouldDisplyOnPrpviding10CharInMobileFildOdRegidtrationPage(RegistrationTest.java:181)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:324)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:74)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:123)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
    </failure>
  </testcase> <!-- ErrorShouldDisplyOnPrpviding10CharInMobileFildOdRegidtrationPage -->
  <testcase name="invalidMobWithLeassDigit" time="47.043" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <testcase name="TestLessDigitinMob" time="48.648" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <testcase name="TestPasseordWithOnlyInt" time="41.818" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <testcase name="testPasswordWithOnlyString" time="42.707" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <testcase name="checkFirstName" time="42.000" classname="com.Tour.qa.testdata.RegistrationTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:568)
at org.testng.Assert.assertEquals(Assert.java:578)
at com.Tour.qa.testdata.RegistrationTest.checkFirstName(RegistrationTest.java:255)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:324)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:74)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:123)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
    </failure>
  </testcase> <!-- checkFirstName -->
  <testcase name="checkLastName" time="42.691" classname="com.Tour.qa.testdata.RegistrationTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:568)
at org.testng.Assert.assertEquals(Assert.java:578)
at com.Tour.qa.testdata.RegistrationTest.checkLastName(RegistrationTest.java:270)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:324)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:74)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:123)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
    </failure>
  </testcase> <!-- checkLastName -->
  <testcase name="checkEmailnotMatch" time="45.703" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <testcase name="BothPasswordFieldNotMatch" time="48.889" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <testcase name="checkMandatoryFieldsOnLeavinfEmpty" time="48.403" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <testcase name="VehiclesFieldWithCharacter" time="42.679" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <testcase name="checkGroupSizeWithString" time="45.849" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <testcase name="HomePageTitleTest" time="35.303" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <testcase name="CheckBanner" time="33.825" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <testcase name="validateGroupText1" time="32.633" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <testcase name="validateGroupSize" time="37.030" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <testcase name="validateChoosFromMenuBar" time="37.193" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <testcase name="validateChoosDateFromMenuBar" time="36.716" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <testcase name="HeadingValidate1" time="35.512" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <testcase name="validateLoginButton" time="40.234" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <testcase name="validateForgot" time="34.284" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <testcase name="ValidateHeading2" time="37.299" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <testcase name="ValidateHeading3" time="31.891" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <testcase name="ValidateHeading4" time="32.801" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <testcase name="ValidateHeading5" time="34.173" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <testcase name="validateService" time="36.826" classname="com.Tour.qa.testdata.RegistrationTest"/>
</testsuite> <!-- com.Tour.qa.testdata.RegistrationTest -->
